python/cudf/cudf/core/column/string.py

```diff
@@ -3537,6 +3533,15 @@ def count(self, pat: str, flags: int = 0) -> SeriesOrIndex:
         >>> index = cudf.Index(['A', 'A', 'Aaba', 'cat'])
         >>> index.str.count('a')
         Int64Index([0, 0, 2, 1], dtype='int64')
+
+        .. pandas-compat::
+            **StringMethods.count**
+
+            -   `flags` parameter currently only supports re.DOTALL
+                and re.MULTILINE.
+            -   Some characters need to be escaped when passing
+                in pat. e.g. ``'$'`` has a special meaning in regex
+                and must be escaped when finding this literal character.
```

Comment:
This second point is not a pandas compatibility note, but rather just a general note about regex matching, can we therefore restore it to the previous notes section.